File: parte001/ex009_salario_empleado.cs
// Se tiene la siguiente información de un empleado: 
// - código del empleado, 
// - nombres, 
// - número de horas trabajadas al mes, 
// - valor hora trabajada, 
// - porcentaje de retención en la fuente.
// Haga un algoritmo que muestre: código, nombres, salario bruto y salario neto.

using System;

namespace Parte001
{
    class Program
    {
        static void Main(string[] args)
        {
            int codigo, horas;
            double valorHora, porcentajeRetencion, salarioBruto, salarioNeto;
            string nombres;

            Console.Write("Ingrese el código del empleado: ");
            codigo = Convert.ToInt32(Console.ReadLine());

            Console.Write("Ingrese los nombres del empleado: ");
            nombres = Console.ReadLine();

            Console.Write("Ingrese el número de horas trabajadas al mes: ");
            horas = Convert.ToInt32(Console.ReadLine());

            Console.Write("Ingrese el valor de la hora trabajada: ");
            valorHora = Convert.ToDouble(Console.ReadLine());

            Console.Write("Ingrese el porcentaje de retención en la fuente: ");
            porcentajeRetencion = Convert.ToDouble(Console.ReadLine());

            salarioBruto = horas * valorHora;
            salarioNeto = salarioBruto - (salarioBruto * porcentajeRetencion / 100);

            Console.WriteLine("Código: " + codigo);
            Console.WriteLine("Nombres: " + nombres);
            Console.WriteLine("Salario bruto: " + salarioBruto);
            Console.WriteLine("Salario neto: " + salarioNeto);

            Console.ReadKey();
        }
    }
}

File: parte001/ex009_salario_empleado.java
// Se tiene la siguiente información de un empleado: 
// - código del empleado, 
// - nombres, 
// - número de horas trabajadas al mes, 
// - valor hora trabajada, 
// - porcentaje de retención en la fuente.
// Haga un algoritmo que muestre: código, nombres, salario bruto y salario neto.

import java.util.Scanner;

public class ex009_salario_empleado {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);
        
        int codigo, horas;
        double valorHora, retencion, salarioBruto, salarioNeto;
        String nombres;
        
        System.out.print("Digite el código del empleado: ");
        codigo = entrada.nextInt();
        entrada.nextLine();
        System.out.print("Digite los nombres del empleado: ");
        nombres = entrada.nextLine();
        System.out.print("Digite el número de horas trabajadas al mes: ");
        horas = entrada.nextInt();
        System.out.print("Digite el valor hora trabajada: ");
        valorHora = entrada.nextDouble();
        System.out.print("Digite el porcentaje de retención en la fuente: ");
        retencion = entrada.nextDouble();
        
        salarioBruto = horas * valorHora;
        salarioNeto = salarioBruto - (salarioBruto * (retencion / 100));
        
        System.out.println("Código: " + codigo);
        System.out.println("Nombres: " + nombres);
        System.out.println("Salario bruto: " + salarioBruto);
        System.out.println("Salario neto: " + salarioNeto);
        
        entrada.close();
    }
}
